linux-user/syscall.c

```diff
@@ -5940,19 +5943,21 @@ static abi_long qemu_execve(char *filename, char *argv[],
     }
 
     new_argp[0] = strdup(qemu_execve_path);
-    new_argp[1] = strdup("-0");
+    /* Propagate the execve behaviour */
+    new_argp[1] = strdup("-execve");
```

Comment:
I realised that this would work fine even if you just assign it like so `new_argp[1] = "-execve"` since the pointers will be valid for the duration of the execve call. Feel free to change it if you want, just a nit